async_simple/coro/test/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 file(GLOB coro_test_src "*.cpp")
+add_compile_options("$<$<CXX_COMPILER_ID:MSVC>:/bigobj>")
```

Comment:
Why do we need this?

Reply:
windows ci failed before,
https://github.com/alibaba/async_simple/actions/runs/8200305183/job/22426819820#step:4:304

Reply:
This shouldn't be related to this patch. Please remove the unrelated change.

Reply:
OK, will create a new PR to add this compile option for msvc.

Reply:
done in this PR: https://github.com/alibaba/async_simple/pull/375